test/jest/unit/cli-monitor-utils.spec.ts

```diff
@@ -155,6 +155,24 @@ describe('cli-monitor-utils test', () => {
     expect(res).toEqual('meta-gradle-project');
   });
 
+  it('getProjectName returns project name when project-name is provided via option', () => {
+    const scannedProject: ScannedProject = {
+      depGraph: {} as any,
+      meta: {},
```

Comment:
You need this to actually test your change:
```suggestion
      meta: {
        gradleProjectName: 'my-gradle-project',
      },
```
Without `meta.gradleProjectName` you would have fallen through to returning `meta['project-name']`. i.e. you haven't run this test without the change and seen it fail.